### ISSUE TITLE
tough-cookie@2.2.2: ReDoS vulnerability

### ISSUE THREAD

**Issue by udf2457** [Azure/azure-xplat-cli#3108]
"npm update -g azure-cli" reports the following:

npm WARN deprecated tough-cookie@2.2.2: ReDoS vulnerability parsing Set-Cookie https://nodesecurity.io/advisories/130

**Comment by haocs** [comment]
```bash
├─┬ azure-arm-website@0.10.0
│ └─┬ azure-common@0.9.12
│   └─┬ request@2.45.0
│     └── tough-cookie@2.3.1 
├─┬ azure-common@0.9.17
│ └─┬ request@2.72.0
│   └── tough-cookie@2.2.2 
├─┬ azure-storage@1.1.0
│ └─┬ request@2.69.0
│   └── tough-cookie@2.2.2 
└─┬ request@2.74.0
  └── tough-cookie@2.3.1 
```
Yes. there are some dependencies using older versions(<2.74) of request library. Need to upgrade `request` to `~2.74.0` to solve this issue.

**Comment by amarzavery** [comment]
@udf2457 - This should be resolved with 0.10.4 release of the CLI, which has already been published
